### ISSUE TITLE
Babel failed to parse the missing semicolon

### ISSUE THREAD

**Issue by dimasyudhatech** [jaw52/node-tool#1]
Hi, Bro! First of all, thank you so much for making such this awesome package to make our life easier especially for those who want to migrate from CRA to Vite. But, I faced the [following issues](https://hastebin.com/share/efoculezoh.ruby) while transforming from ".js" to ".jsx". Any idea regarding such those issues? Thanks a buch :)

**Comment by jaw52** [comment]
Thank you for using･◡･

I have preset some babel settings, but they have not covered your project

## Install
Install the beta version of the tool first

```
npx @jaw52/transform-jsx-for-vite@1.8.0-beta.2
```

## solve
In the new beta version I added` quick mode`，you can try it，But it is not guaranteed to be more accurate than babel in identifying jsx

If you want to identify more accurately，need you to choose `precise mode`,You can follow the steps below to help me improve the library

You can find out the grammar that fails to parse according to the number of lines in the prompt, and you can provide me with a minimal demo

Or can you show me your babel settings

In short, thank you for providing this issue

**Comment by dimasyudhatech** [comment]
First of all, thank you so much for responding my issue. After trying to install the beta one, it seems that everything is working flawlessly. Below is the output

![image](https://user-images.githubusercontent.com/64878541/225586181-ba776160-8ad9-4287-b5ee-37aa76cf2679.png)

Thanks a million :)
